alacritty/src/config/monitor.rs

```diff
@@ -10,7 +10,7 @@ use alacritty_terminal::util;
 use crate::event::EventProxy;
 
 pub struct Monitor {
-    _thread: ::std::thread::JoinHandle<()>,
+    _thread: std::thread::JoinHandle<()>,
```

Comment:
If these are touched, we might as well properly import `std::thread` at the top of the file.